src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -236,8 +237,27 @@ function getTableRowClass(gauge: VotingGaugeWithVotes): string {
         </div>
       </template>
       <template #nextPeriodVotesCell="gauge">
-        <div v-if="!isLoading" class="py-4 px-6 text-right">
+        <div v-if="!isLoading" class="flex justify-end py-4 px-6">
           <GaugeVoteInfo :gauge="gauge" />
+          <IconLimit
+            v-if="gauge.pool.symbol === 'veBAL'"
+            size="sm"
+            amount="10"
+            :tooltip="
+              $t('veBAL.liquidityMining.limitsTooltip.distributionsCappedAt10%')
+            "
+            class="ml-1"
+          />
+          <IconLimit
+            v-if="gauge.relativeWeightCap > 0"
```

Comment:
This is a rough initial implementation. Ideally we should check that relativeWeightCap is not null and use its value to determine what is displayed in the icon.

Reply:
Ok, I made some changes to this effect.

Reply:
lgtm, thanks